doc/user/pbr.rst

```diff
@@ -96,7 +96,7 @@ end destination.
    Use the nexthop-group NAME as the place to forward packets when the match
    commands have matched a packet.
 
-.. clicmd:: set nexthop [A.B.C.D|X:X::X:XX] [interface] [nexthop-vrf NAME]
+.. clicmd:: set nexthop {<A.B.C.D|X:X::X:XX>|interface} [nexthop-vrf NAME]
```

Comment:
This just seems wrong again.

Reply:
Please see my previous comments about this.